point_pattern.py

```diff
@@ -27,10 +27,11 @@ def manhattan_distance(a, b):
     distance : float
                The Manhattan distance between the two points
     """
-    distance =  None # Add the algorithm to compute manhattan distance here
+    distance = ((a[0]-b[0])+((a[1]-b[1])))
```

Comment:
Here you have a few options.  The algorithm is |x1 - x2| + |y1 - y2|.  You got the point ordering and operators in the correct place.  This just needs to compute `abs((a[0]-b[0]) + abs(a[1]-b[1])`